src/Cake.Issues/IIssueComparer.cs

```diff
@@ -117,6 +118,7 @@ public int GetHashCode(IIssue obj)
                         obj.PriorityName,
                         obj.Rule,
                         obj.RuleUrl,
+                        obj.Run,
```

Comment:
obj.run should also included in true case of this.compareOnlyPersistentProperties condition.